integration/live.yml

```diff
@@ -0,0 +1,56 @@
+---
+storage:
+  dropbox:
+    specs_enabled: true
+    api_token: qlv6P0K5_KAAAAAAAAAwmTZY-GE45tpYiA3gZD1-sn4DlB1qJltrnTpUXC0qo3OG
+    # these may be changed if needed
+    path: backup_testing
+  s3:
+    specs_enabled: false
+    access_key_id: <your key>
+    secret_access_key: <your secret>
+    region: <your region>
+    bucket: <your bucket>
+    # change if needed
+    path: backup_testing
+  cloudfiles:
+    specs_enabled: false
+    username: <your username>
+    api_key: <your key>
+    auth_url: # if needed
+    region: # if needed
+    servicenet: false
+    container: <your container>
+    segments_container: <another container>
+    path: backup_testing
+syncer:
+  cloud:
+    s3:
+      specs_enabled: false
+      access_key_id: <your key>
+      secret_access_key: <your secret>
+      region: <your region>
+      bucket: <your bucket>
+      # change if needed
+      path: backup_testing
+    cloudfiles:
+      specs_enabled: false
+      username: <your username>
+      api_key: <your key>
+      auth_url: # if needed
+      region: # if needed
+      servicenet: false
+      container: <your container>
+      path: backup_testing
+notifier:
+  mail:
+    specs_enabled: false
+    from: <from email>
+    to: <to email>
+    address: <your address>
+    port: <your port>
+    user_name: <your user_name>
+    password: <your password>
+    authentication: :plain
+    encryption: :starttls
+    openssl_verify_mode:
```

Comment:
I think this whole file can be removed and put in the `.gitignore` file. Instead, devs should use the `integration/live.yml.template` file, right?